docs/doc/13-monitor/prometheus-and-granafa.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Prometheus & Grafana
+---
+
+[Prometheus](https://prometheus.io/) is a time series database that features a flexible query language and a multi-dimensional data model.
+
+[Grafana](https://grafana.com/grafana) is an open-source tool used for analyzing and visualizing metrics.
+
+Databend supports using Prometheus to store monitoring and performance metrics, which can be visualized through Grafana.
+
+## Tutorial: Monitor Databend with Prometheus & Grafana
+
+### Step 1. Deploy Databend
+
+Follow the [Deployment Guide](https://databend.rs/doc/deploy) to deploy Databend.
```

Comment:
Follow the [Deployment Guide](https://databend.rs/doc/deploy) to deploy a standalone Databend.